.changeset/curvy-suns-roll.md

```diff
@@ -0,0 +1,7 @@
+---
+'@clerk/backend': patch
+---
+
+Remove createEmail function from @clerk/backend.
+
+The corresponding endpoint is also being removed from the API.
```

Comment:
Discussed with @Nikpolik offline and we will not be providing an alternative method for the time being. 

Can we please add a note informing any customers currently using this to reach out to our support team if they wish to update to v5?

```suggestion
The `emails` endpoint helper and the corresponding `createEmail` method have been removed from the `@clerk/backend` SDK and `apiClint.emails.createEmail` will no longer be available. We will not be providing an alternative method for creating and sending emails directly from our JavaScript SDKs with this release. If you are currently using `createEmail` and you wish to update to the latest SDK version, please reach out to our support team (https://clerk.com/support) so we can assist you.
```

Reply:
Much better message thanks!